### DISCUSSION TITLE
WorldMultiplay 파일 오류

### DISCUSSION THREAD

**Discussion by dr-turt1e** [naverz/zepeto-studio-kor#499] [5. Q&A]
완전 초보라서 무슨말인지는 모르겟지만 유튜브 보고 따라 했습니다.

WorldMultiplay 파일은 다운 받아서 실행 했던 것 같습니다.
![image](https://user-images.githubusercontent.com/101694731/158528518-7c3e9d49-58e1-48eb-bb26-372d361a18a6.png)
4번째 줄에 Vector3가 아닌 Vertor3로 되어 있어 
index,ts와 ClientStater에서 모두 Vertor3로 기입해야 플레이를 하면 아바타가 나옵니다.
Vector3로 수정했더니 오류가 생겨서 샘플 파일이 잘못 된것 아닐까 문의 드립니다.

Vertor3때문인지는 모르겠지만 플레이시에는 아바타가 잘 나오는데
QR코드로 핸드폰으로 접속하면 아바타가 나오지 않습니다.
![image](https://user-images.githubusercontent.com/101694731/158529600-d2c96dce-0e33-42ad-97b4-3669c124404e.png)

유니티를 사용해 제작하는 것은 너무 어렵고 정보도 별로 없네요.
그래도 유튜브 동영상 만들어주셔서 아바타가 화면에 나오기는 했어요..ㅠㅜ
답변 기다리겠습니다

**Comment by minjujuu** [comment]
schemas.json을 직접 편집하시기보단, 영상에서처럼 유니티 인스펙터 창에서 편집하시는 것을 권장드립니다.
World.Multiplay 폴더에 있는 Schemas를 편집하실 때 다음 이미지처럼 되어있는지 확인해봐주세요!

다음 이미지처럼 되어 있을 경우 index.ts와 ClientStarter에서 Vertor3로 수정하지 않으셔도 정상 동작할 것으로 보입니다.

![image](https://user-images.githubusercontent.com/79794414/158537075-123a8880-a898-4342-9c92-efc202c753e6.png)

**Reply by dr-turt1e** [reply]
답변 감사합니다 :)
인스펙터 창에도 잘 되어있고 혹시나해서 다른 이름으로 수정했다가 바꿔도 보았는데 계속 오류가 생겨 결국 다시 만들었습니다ㅜㅠ
